Formula/h2.rb

```diff
@@ -1,9 +1,10 @@
 class H2 < Formula
-  desc "Java SQL database"
-  homepage "https://www.h2database.com/"
-  url "https://www.h2database.com/h2-2018-03-18.zip"
-  version "1.4.197"
-  sha256 "a45e7824b4f54f5d9d65fb89f22e1e75ecadb15ea4dcf8c5d432b80af59ea759"
+  desc "H2 is an embeddable RDBMS written in Java."
+  homepage "https://h2database.com"
```

Comment:
No reason to modify the description or homepage

Reply:
I feel like description would be more accurate in terms of two things:
- embeddable RDBMS instead of just a DB
- DB in java instead of plain DB description.


Yeah, the `homepage` change is not quite needed but, I did adopt it from the github tagline. I can switch it back on it.

Reply:
I agree with FX here. Original description better matches what's on the website.
Both variants of the URL work. Please revert them back, and remove `head`.

Reply:
Sure.